Packs/DelineaDSV/pack_metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "DelineaDSV",
+    "description": "This integration pack helps you retrieve the data stored in the Delinea DevOps Storage Vault and use it in other integrations.",
+    "support": "partner",
+    "currentVersion": "2.0.0",

```

Comment:
Please change to `1.0.0`, as our build process expects new packs to always start with `1.0.0`.